iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/provider/TestConfigBasedIcebergTableOpsProvider.java

```diff
@@ -51,13 +47,11 @@ public void testValidIcebergTableOps() {
     // jdbc backend catalog
     config.put("catalog.jdbc_backend.catalog-backend-name", jdbcCatalogName);
     config.put("catalog.jdbc_backend.catalog-backend", "jdbc");
-    config.put(
-        "catalog.jdbc_backend.uri",
-        String.format("jdbc:h2:%s;DB_CLOSE_DELAY=-1;MODE=MYSQL", STORE_PATH));
+    config.put("catalog.jdbc_backend.uri", "jdbc:sqlite::memory:");
     config.put("catalog.jdbc_backend.warehouse", "/tmp/usr/jdbc/warehouse");
     config.put("catalog.jdbc_backend.jdbc.password", "gravitino");
     config.put("catalog.jdbc_backend.jdbc.user", "gravitino");
-    config.put("catalog.jdbc_backend.jdbc-driver", "org.h2.Driver");
+    config.put("catalog.jdbc_backend.jdbc-driver", "org.sqlite.JDBC");
```

Comment:
Why do we use Sqlite instead of `H2`?

Reply:
no special reason, it has sqlite dependence